demisto_sdk/validation/file_validator.py

```diff
@@ -536,16 +553,22 @@ def is_valid_structure(self):
         if self.use_git:
             if self.branch_name != 'master' and (not self.branch_name.startswith('19.') and
                                                  not self.branch_name.startswith('20.')):
-                print("Validates only committed files")
+                print('Validates only committed files')
                 self.validate_committed_files()
                 self.validate_against_previous_version(no_error=True)
             else:
                 self.validate_against_previous_version(no_error=True)
-                print("Validates all of Content repo directories according to their schemas")
+                print('Validates all of Content repo directories according to their schemas')
                 self.validate_all_files()
         else:
-            print("No using git, validating all files")
-            self.validate_all_files()
+            if self.file_path:
+                branches = run_command('git branch')
+                branch_name_reg = re.search(r'\* (.*)', branches)
+                self.branch_name = branch_name_reg.group(1)
+                self.validate_committed_files()
+            else:
+                print('No using git, validating all files')
+                self.validate_all_files()
```

Comment:
could you explain this logic

Reply:
when the user run the specific file validator, without the `use_git` flag, the logic of receiving the working branch does not happen. So, to verify against master I need to fetch it's branch.
The `else` if for the regular validate without use_git.

Reply:
But, it would be smarter to make it into a function, so Im fixing that.